testing/test_assertrewrite.py

```diff
@@ -65,18 +65,33 @@ def getmsg(f, extra_ns=None, must_pass=False):
             pytest.fail("function didn't raise at all")
 
 
+def adjust_body_for_new_docstring_in_module_node(m):
+    """Module docstrings in 3.8 are part of Module node.
+    This was briefly in 3.7 as well but got reverted in beta 5.
+
+    It's not in the body so we remove it so the following body items have
+    the same indexes on all Python versions:
+
+    TODO:
+
+    We have a complicated sys.version_info if in here to ease testing on
+    various Python 3.7 versions, but we should remove the 3.7 check after
+    3.7 is released as stable to make this check more straightforward.
+    """
+    if sys.version_info < (3, 8) or (3, 7) <= sys.version_info <= (3, 7, 0, "beta", 4):
+        assert len(m.body) > 1
+        assert isinstance(m.body[0], ast.Expr)
+        assert isinstance(m.body[0].value, ast.Str)
+        del m.body[0]
+    return m
```

Comment:
since this is a mutating helper, i propose not having a return value

Reply:
just do it in place? ok

Reply:
Done.